lib/importproject.cpp

```diff
@@ -1022,9 +1022,12 @@ bool ImportProject::importCppcheckGuiProject(std::istream &istr, Settings *setti
             temp.addons = readXmlStringList(node, "", CppcheckXml::AddonElementName, nullptr);
         else if (strcmp(node->Name(), CppcheckXml::TagsElementName) == 0)
             node->Attribute(CppcheckXml::TagElementName); // FIXME: Write some warning
-        else if (strcmp(node->Name(), CppcheckXml::ToolsElementName) == 0)
-            node->Attribute(CppcheckXml::ToolElementName); // FIXME: Write some warning
-        else if (strcmp(node->Name(), CppcheckXml::CheckHeadersElementName) == 0)
+        else if (strcmp(node->Name(), CppcheckXml::ToolsElementName) == 0) {
+            std::list<std::string> toolList = readXmlStringList(node, "", CppcheckXml::ToolElementName, nullptr);
+            for(std::string toolName : toolList)
```

Comment:
please use const reference to avoid datacopy.

Reply:
Some nits:
please use const reference to avoid data copy.
I'd like that you put `{}` around this for loop body I just think it looks a bit weird with nested `for` and `if` without `{}`.